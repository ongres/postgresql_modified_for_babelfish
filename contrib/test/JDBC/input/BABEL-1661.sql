-- BABEL-1661
-- Test CONVERT to VARCHAR applies typmod
SELECT CONVERT(VARCHAR(10), SERVERPROPERTY('productversion'));
GO

SELECT DATALENGTH(CONVERT(VARCHAR(10), SERVERPROPERTY('productversion')));
GO

SELECT CONVERT(VARCHAR(2), SERVERPROPERTY('productversion'));
GO

SELECT DATALENGTH(CONVERT(VARCHAR(2), SERVERPROPERTY('productversion')));
GO
