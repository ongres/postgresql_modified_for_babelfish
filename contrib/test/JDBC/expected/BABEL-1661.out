-- BABEL-1661
-- Test CONVERT to VARCHAR applies typmod
SELECT CONVERT(VARCHAR(10), SERVERPROPERTY('productversion'));
GO
~~START~~
varchar
12.0.2000.
~~END~~


SELECT DATALENGTH(CONVERT(VARCHAR(10), SERVERPROPERTY('productversion')));
GO
~~START~~
int
10
~~END~~


SELECT CONVERT(VARCHAR(2), SERVERPROPERTY('productversion'));
GO
~~START~~
varchar
12
~~END~~


SELECT DATALENGTH(CONVERT(VARCHAR(2), SERVERPROPERTY('productversion')));
GO
~~START~~
int
2
~~END~~

